lambdas/src/apis/collections/controllers/collections.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from 'express'
+import { SmartContentServerFetcher } from '../../../SmartContentServerFetcher'
+
+export async function getStandardErc721(fetcher: SmartContentServerFetcher, req: Request, res: Response) {
+    // Method: GET
+    // Path: /standard/erc721/:contract/:option/:emission
+    const { contract, option, emission } = req.params;
+
+    try {
+        const entities:WearableEntity[] = await fetcher.fetchJsonFromContentServer(`/entities/wearable?pointer=${contract}-${option}`)
+        if (entities && entities.length > 0 && entities[0].metadata) {
+            const wearableMetadata = entities[0].metadata
+            const id = `dcl://${contract}/${option}`
+            const name = wearableMetadata.i18n[0]?.text // TODO: should we ensure it's the english description?
+            const totalEmission = RARITIES_EMISSIONS[wearableMetadata.rarity]
+            const description =  emission ? `DCL Wearable ${emission}/${totalEmission}` : ''
+            const image = fetcher.getExternalContentServerUrl() + '/contents/' + wearableMetadata.image
+            const thumbnail = fetcher.getExternalContentServerUrl() + '/contents/' + wearableMetadata.thumbnail
+            const standardErc721 = {
+                id,
+                name,
+                description,
+                language: "en-US",
+                image,
+                thumbnail
+            }
+            res.send(standardErc721)
+        } else {
+            res.status(404).send()
+        }
+    } catch(e) {
+        res.status(500).send(e.messsge);
+    }
+}
+
+const RARITIES_EMISSIONS = {
+    "common"   : 100000,
+    "uncommon" : 10000,
+    "rare"     : 5000,
+    "epic"     : 1000,
+    "legendary": 100,
+    "mythic"   : 10,
+    "unique"   : 1
+}
+
+type WearableEntity = {
+    id: string
+    type: string
+    pointers: string[]
+    timestamp: number
+    content?: EntityContent[]
+    metadata?: WearableMetadata
+}
+
+type EntityContent = {
+    file: string,
+    hash: string,
+}
+
+type WearableMetadata = {
```

Comment:
Maybe we should get someone with more experience with wearables to validate this

Reply:
This is the current structure for wearables as they are in: https://wearable-api.decentraland.org/v2/addresses/0x87956abC4078a0Cc3b89b419928b857B8AF826Ed/wearables